modules/markup/markdown/markdown.go

```diff
@@ -124,36 +124,35 @@ func (r *Renderer) Image(out *bytes.Buffer, link []byte, title []byte, alt []byt
 
 const (
 	blackfridayExtensions = 0 |
-		blackfriday.EXTENSION_NO_INTRA_EMPHASIS |
-		blackfriday.EXTENSION_TABLES |
-		blackfriday.EXTENSION_FENCED_CODE |
-		blackfriday.EXTENSION_STRIKETHROUGH |
-		blackfriday.EXTENSION_NO_EMPTY_LINE_BEFORE_BLOCK |
-		blackfriday.EXTENSION_DEFINITION_LISTS |
-		blackfriday.EXTENSION_FOOTNOTES |
-		blackfriday.EXTENSION_HEADER_IDS |
-		blackfriday.EXTENSION_AUTO_HEADER_IDS
+		blackfriday.NoIntraEmphasis |
+		blackfriday.Tables |
+		blackfriday.FencedCode |
+		blackfriday.Strikethrough |
+		blackfriday.NoEmptyLineBeforeBlock |
+		blackfriday.DefinitionLists |
+		blackfriday.Footnotes |
+		blackfriday.HeadingIDs |
+		blackfriday.AutoHeadingIDs
 	blackfridayHTMLFlags = 0 |
-		blackfriday.HTML_SKIP_STYLE |
-		blackfriday.HTML_OMIT_CONTENTS |
-		blackfriday.HTML_USE_SMARTYPANTS
+		blackfriday.SkipHTML |
+		blackfriday.Smartypants
```

Comment:
`OmitContents`?

Reply:
It appears `blackfriday.HTML_OMIT_CONTENTS` was sunset without a direct replacement, it may be that some other flag or option within the `blackfriday.HTMLRenderer` struct may replace this.